### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by cs-renuka-andhale**

### CODE REVIEW

File: info.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "name": "serviceNowSecurityIncidentResponseIntegration",
+    "version": "1.0.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "ServiceNow Security Incident Response Integration",
+    "dependencies": [],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "description": "With the ServiceNow Security Incident Response (SIR) module integration Solution pack for FortiSOAR , users can manage the life cycle of your security incidents from initial analysis to containment, eradication, and recovery. Security Incident Response enables you to get a comprehensive understanding of incident response procedures performed by your analysts, and understand trends and bottlenecks in those procedures with analytic-driven dashboards and reporting.",
+    "help": null,
```
Comment (cs-rohit-khune) [inline info.json]: @cs-renuka-andhale please add document  link

File: info.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{
+    "name": "serviceNowSecurityIncidentResponseIntegration",
+    "version": "1.0.0",
+    "type": "solutionpack",
+    "local": true,
+    "label": "ServiceNow Security Incident Response Integration",
+    "dependencies": [],
```
Comment (cs-rohit-khune) [inline info.json]: @cs-renuka-andhale Please add "SOAR Framework" as a dependency SP

**Review by cs-rohit-khune — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+# Overview
+
+**ServiceNow Security Incident Response (SIR) integration Solution pack** helps users to manage the life cycle of your security incidents from initial analysis to containment, eradication, and recovery. Security Incident Response enables you to get a comprehensive understanding of incident response procedures performed by your analysts, and understand trends and bottlenecks in those procedures with analytic-driven dashboards and reporting.
+
+You can configure the ServiceNow connector included with this solution pack to ingest **Security Incidents** and create in FortiSOAR.
+
+# Next Steps 
+ 
+| [Installation](https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/develop/docs/setup.md#installation) | [Configuration](https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/develop/docs/setup.md#configuration) | [Usage](https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/develop/docs/usage.md) | [Contents](https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/develop/docs/contents.md) |
```
Comment (cs-suyog-jinturkar) [inline README.md]: doc links are pointing to GDPR repo. @cs-rohit-khune @cs-renuka-andhale

File: info.json [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+            "name": "sOARFramework",
+            "type": "solutionpack",
+            "label": "SOAR Framework",
+            "version": "1.1.0",
+            "minVersion": null
+        }
+    ],
+    "prerequisite": null,
+    "publisher": "Fortinet",
+    "description": "With the ServiceNow Security Incident Response (SIR) module integration Solution pack for FortiSOAR , users can manage the life cycle of your security incidents from initial analysis to containment, eradication, and recovery. Security Incident Response enables you to get a comprehensive understanding of incident response procedures performed by your analysts, and understand trends and bottlenecks in those procedures with analytic-driven dashboards and reporting.",
```
Comment (cs-suyog-jinturkar) [inline info.json:19]: Please review this description text with Abhishek S

File: info.json [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-servicenow-security-incident-response-integration/blob/release/1.0.0/README.md",
```
Comment (cs-suyog-jinturkar) [inline info.json:20]: Please create release/1.0.0 branch if its not created as of now. This PR is also raised on same branch.

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comments

File: info.json (hunk 5 above)
Comment (cs-renuka-andhale) [reply info.json:20]: @cs-suyog-jinturkar I don't have permissions to create new branch, can you please create it.

**Comment by cs-renuka-andhale** [comment]
closing the PR

**Pull request closed without merging**